### ISSUE TITLE
New release breaks precss

### ISSUE THREAD

**Issue by Jimexist** [simonsmith/postcss-property-lookup#24]
I am getting this error on the new release:

```
/home/jiayu/iffedu/node_modules/precss/index.js:104
			instance.use(processor.plugin(processorOptions));
			                       ^

TypeError: processor.plugin is not a function
    at /home/jiayu/iffedu/node_modules/precss/index.js:104:27
    at Array.forEach (native)
    at /home/jiayu/iffedu/node_modules/precss/index.js:91:13
    at creator (/home/jiayu/iffedu/node_modules/postcss/lib/postcss.js:60:39)
    at Function.postcss.plugin (/home/jiayu/iffedu/node_modules/postcss/lib/postcss.js:66:23)
    at Object.<anonymous> (/home/jiayu/iffedu/node_modules/precss/index.js:85:26)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
```
and everything works fine when i switch back to `1.1.3`.

**Comment by lennerd** [comment]
The problem is the use of Babel. `build/index.js` should return a function but instead returns a ES6 babel module object.

**Comment by simonsmith** [comment]
Just fixing this now

**Comment by simonsmith** [comment]
Fixed in 1.2.1, apologies for any disruption.

**Comment by lennerd** [comment]
Thank you for the quick fix!
